troll-o-matic.py

```diff
@@ -150,6 +155,7 @@ def do_review(self, review_type, change, fixes_ref, msg, notify, vote):
       print('------')
 
   def handle_successful_review(self, change, prefix, fixes_ref):
+    # TODO: We should tag FROMLIST: BACKPORT: patches as incorrect, if needed
```

Comment:
I've kind of blissfully ignored double-prefixed patches until now, but I guess we can't do that if we support FROMLIST :)